packages/react-table/src/components/Table/ActionsColumn.tsx

```diff
@@ -30,6 +30,8 @@ export interface ActionsColumnProps extends Omit<React.HTMLProps<HTMLElement>, '
   innerRef?: React.Ref<any>;
   /** Ref to forward to the first item in the popup menu */
   firstActionItemRef?: React.Ref<HTMLButtonElement>;
+  /** Flag indicating that the dropdown's onOpenChange callback should not be called. */
+  isOnOpenChangeDisabled?: boolean;
```

Comment:
Per above I've added this prop. From what I can tell since ActionsColumn has its own isOpen state, being able to pass a custom onOpenChange callback wouldn't be as easy without introducing other props to customize things, and at that point I'd wonder if it wouldjust make more sense to allow passing a custom Dropdown itself (rather than right now we only really allow a custom `actionsToggle`)